### DISCUSSION TITLE
Can I use this to move my user data to a domain account on the same computer?

### DISCUSSION THREAD

**Discussion by tedd-wq** [belowaverage-org/SuperGrate#56] [Q&A] [answered]
Also, how do I use this to copy the folder redirections I have for Documents, Downloads, Pictures, Photos, Video and Music, but *not* their contents?

**Comment by krisdb2009** [comment]
Hi @tedd-wq ,

Super Grate wraps around Microsoft's USMT.
If you want to move your account to a domain account, you can do so by appending a custom "LoadStateParameter" in the SuperGrate settings menu.

![image](https://user-images.githubusercontent.com/8226974/107973419-31b0c780-6f83-11eb-9952-3c008420e324.png)

A list of parameters is available here: https://docs.microsoft.com/en-us/windows/deployment/usmt/usmt-loadstate-syntax

The specific option you likely want to use is `/mu:OldDomain<em>OldUserName:[NewDomain]NewUserName`

For the folder redirections question, your best bet is to exclude these directories entirely. To accomplish this, browse to the MigUser.xml under ./USMT/X64 and comment out the XML options indicating to migrate the folders you want to exclude.

![image](https://user-images.githubusercontent.com/8226974/107973712-8f451400-6f83-11eb-862f-b95d250b662e.png)


I hope this helps!

**Comment by krisdb2009** [comment, marked as answer]
Hi everyone,

I have an update to this question:

If you want to restore a user profile onto another domain or as a different user name, follow the steps below.

1. After performing a backup like normal, click "List Store".
2. Right click the user you want to rename.
3. Select "Set destination user name..."
4. In the dialog, enter the new username or domain and username e.g:

%DOMAIN%\%USERNAME% or %USERNAME% ➡ BELOWAVERAGE\dylan or dylan

![image](https://user-images.githubusercontent.com/8226974/138915283-7539b16c-ad12-4afd-b785-be28efb87a0e.png)
![image](https://user-images.githubusercontent.com/8226974/138915301-492ba55b-310b-4472-8d77-9c13ede48060.png)
